src/components/shared/Map/MapPointer.tsx

```diff
@@ -31,8 +62,8 @@ function MapPointer({
     <Circle
       center={center}
       radius={radius}
-      fillColor={color}
-      color={color}
+      fillColor={colorCircle(sensorData, 'PM10')}
+      color={colorCircle(sensorData, 'PM10')}
```

Comment:
You can assign the resulting color to a variable to prvent code repetition. Create a variable before the return statement and replace `colorCircle(sensorData, 'PM10')` with this variable.